Session9/src/com/itschool/homework/Session9Challenge5.java

```diff
@@ -0,0 +1,18 @@
+package com.itschool.homework;
+
+import java.time.LocalDate;
+
+public class Session9Challenge5 {
+    public static void main(String[] args) {
+        Session9Challenge5 run = new Session9Challenge5();
+        run.isTodaySpecificDate();
+    }
+
+    public void isTodaySpecificDate() {
```

Comment:
fine, but it would have been nice that this method returns a `boolean` (like its name suggests) instead of `void`.